### ISSUE TITLE
Ability to modify parentId from a policy

### ISSUE THREAD

**Issue by vronin** [balderdashy/sails#2216]
Currently, blueprints/actions/add.js and remove.js does "req.param('parentid');". As result, it's not modifiable from a policy.

I think, it should to actionUtil.requireParentPk(req) and in requireParentPk it should do something similar to parsePk:

 var pk = req.options.id || (req.options.where && req.options.where.id) || req.param('id');

**Comment by cyrilchapon** [comment]
Definately.
As a use-case, consider the following:

I'm trying to implement a custom route

```
'POST /bean/byaddress/:address/locations' : [{
  policy: 'beanChangeAddressToId'
}, {
  blueprint: 'add',
  model: 'bean',
  alias: 'locations'
}]
```

I'd like to turn address to a real id, but that's just absolutely not achievable with that `req.param('parentid');` without adding an express middleware

**Comment by sailsbot** [comment]
Thanks for posting, @vronin.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
